playbooks/roles/shadowsocks/tasks/main.yml

```diff
@@ -17,27 +11,23 @@
   command: cat {{ shadowsocks_password_file }}
   register: shadowsocks_password
 
-- name: Install Shadowsocks
-  pip: name=shadowsocks
+- name: Add the Shadowsocks-libev APT key
+  apt_key: url=http://shadowsocks.org/debian/1D27208A.gpg state=present
+  
+- name: Add the Shadowsocks-libev repository
+  apt_repository: repo='deb http://shadowsocks.org/ubuntu trusty main' state=present
+
+- name: Install shadowsocks-libev
+  apt: name=shadowsocks-libev
 
 - name: Generate Shadowsocks config file
   template: src=config.json.j2
             dest={{ shadowsocks_location }}/config.json
-            owner=shadowsocks
-            group=shadowsocks
             mode=640
-  notify: Restart Shadowsocks
-
-- name: Generate Shadowsocks init script
-  template: src=shadowsocks-init.j2
-            dest=/etc/init.d/shadowsocks
-            owner=root
-            group=root
-            mode=755
 
-- name: Enable the Shadowsocks init script so the service will start at boot
-  command: update-rc.d shadowsocks defaults
-           creates=/etc/rc0.d/K01shadowsocks
+- name: Symlink config to /etc/ directory
+  file: src={{ shadowsocks_location }}/config.json dest=/etc/shadowsocks-libev/config.json state=link force=yes
```

Comment:
Another convention within the Streisand repository is to place all module arguments on separate lines with proper indentation for maximum readability.

For example, instead of this:
```
file: src={{ shadowsocks_location }}/config.json dest=/etc/shadowsocks-libev/config.json state=link force=yes
```

We do this:
```
file: src={{ shadowsocks_location }}/config.json
      dest=/etc/shadowsocks-libev/config.json
      state=link
      force=yes
```